RecoBTag/TensorFlow/plugins/DeepDoubleXTFJetTagsProducer.cc

```diff
@@ -150,10 +151,63 @@ void DeepDoubleBTFJetTagsProducer::fillDescriptions(edm::ConfigurationDescriptio
   desc.add<unsigned int>("nThreads", 1);
   desc.add<std::string>("singleThreadPool", "no_threads");
 
-  descriptions.add("pfDeepDoubleBJetTags", desc);
+  descriptions.add("pfDeepDoubleBvLJetTags", desc);
+
+  // pfDeepDoubleCvLJetTags
+  edm::ParameterSetDescription desc2;
+  desc2.add<edm::InputTag>("src", edm::InputTag("pfDeepDoubleXTagInfos"));
+  desc2.add<std::vector<std::string>>("input_names", 
+    { "input_1", "input_2", "input_3" });
+  desc2.add<edm::FileInPath>("graph_path",
+    edm::FileInPath("RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDC.pb"));
+  desc2.add<std::vector<std::string>>("lp_names",
+    { "db_input_batchnorm/keras_learning_phase" });
+  desc2.add<std::vector<std::string>>("output_names",
+    { "ID_pred/Softmax" });
+  {
+    edm::ParameterSetDescription psd0;
+    psd0.add<std::vector<unsigned int>>("probQCD", {0});
+    psd0.add<std::vector<unsigned int>>("probHcc", {1});
+    psd0.addOptional<std::vector<unsigned int>>("probHbb");
```

Comment:
The `probQCD` and `probHcc` should be marked as optional as well (but keeping the default values). Otherwise, if they are missing, they will be injected in the configuration during the validation step, which, I believe, is not what you want (but I could be wrong).